.theia/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "cpp.buildConfigurations": [
+        {
+            "name": "",
+            "directory": ""
```

Comment:
Are these necessary?

Reply:
One of them are needed otherwise I was able to debug. I removed the second entry.